### PR TITLE
Overview Module: Add dummy components

### PR DESCRIPTION
**Pull request by BirdHighway**
Overview: Create placeholder components for major elements of the overview module.
References ticket #97

### CODE REVIEW

**Review by kalebal — APPROVED** [review]
Looks good!

File: client/src/overview/AddToCart.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const AddToCart = () => {
+  return (
+    <h1>Add to Cart</h1>
+  );
+};
+
+export default AddToCart;
```
Comment (kalebal) [inline client/src/overview/AddToCart.jsx:9]: Does this file need a new line at the end?

File: client/src/overview/StyleSelector.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const StyleSelector = () => {
+  return (
+    <h3>Style Selector</h3>
+  );
+};
+
+export default StyleSelector;
```
Comment (kalebal) [inline client/src/overview/StyleSelector.jsx:9]: Needs a new line

File: client/src/overview/ProductDetail.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    <div>
+      <h4>Stars</h4>
+      <h4>Category</h4>
+      <h1>Product Name</h1>
+      <h4>Price</h4>
+    </div>
+  );
+};
+
+export default ProductDetail;
```
Comment (kalebal) [inline client/src/overview/ProductDetail.jsx:14]: Needs a new line

**Review by kalebal — CHANGES_REQUESTED** [review]
AddToCart, ImageGallery, ProductDescription, ProductDetail, and StyleSelector need new lines at the end of the file 
(sorry I approved and then saw the other files had new lines -- still figuring this out, my b)

**Review by philteves — APPROVED** [review]
love those newlines, chief!

**Review by kalebal — APPROVED** [review]
Nice lines!

**Pull request merged**